dockerfiles/packaging/debian/package/etc/james/jmap.properties

```diff
@@ -0,0 +1,13 @@
+# Configuration file for JMAP
+
+enabled=true
+
+tls.keystoreURL=file://conf/keystore
```

Comment:
I see you are distributing a default keystore. I think we should not do that and create one at installation time, no?